04_documentation/code_documentation_tools_exercise.md

```diff
@@ -0,0 +1,92 @@
+
+    - Verify that the "Pages" feature of your repository has been activated. When you are on your repository on GitHub go to "Settings" (at top of window) -> "Pages". It should show an information box " Your site is ready to be published at URL" with the URL to the website of your documentation.
+
+      If the "Pages" feature has not been activated automatically, choose the `gh-pages` branch as source and save/activate the website.
+
+      Check the website. If you get a "404" error
+
+      ```text
+      404
+
+      There isn't a GitHub Pages site here.
+      ```
+
+      wait for 5-10 minutes and try again. It takes a while for the website to become visible.
+
+    - Add the URL of the documentation to your GitHub project. Go to your repository on GitHub and edit the "About" section at the top-right corner. For editing clock on the small gear symbol. Add the URL in the box "Website".
+- Verify that your website looks as expected and that the "Edit on GitHub" feature works for all pages.
+- Submit your documentation website via an issue. You cannot submit a pull request since you cannot use a fork in the exercise. You can also add the issue before you are finished in order to ask question. In that case please mention us in the issue such that we get notified. Please write a short note in the issue when you are done and the issue is ready to review. **TODO** Skip this for testing the exercise.
+    - Create an issue in the [`code-documentation` repository](https://github.com/Simulation-Software-Engineering/documentation-tools-exercise).
+    - The issue request should be named `[USERNAME] MkDocs exercise`, e.g., `[jaustar] MkDocs exercise`. Please use the GitLab username here.
+    - Please add a link to your GitHub repository and also a link to your website in the issue.
+
+## Optional Tasks
+
+- [Change the theme](https://www.mkdocs.org/user-guide/choosing-your-theme/#choosing-your-theme) of your documentation. Choose either an integrated theme or a third party theme and/or configure the current scheme differently. Please mention what theme and what settings you have used.
+- Add a [GitHub action](https://github.com/marketplace/actions/deploy-mkdocs) to automatically deploy your documentation every time a change is pushed to the `main` branch.
+- Instead of (only) hosting the documentation on GitHub Pages we can also host the documentation on [Read the Docs](https://readthedocs.org/). You have to make sure to configure [Read the Docs](https://docs.readthedocs.io/en/stable/config-file/v2.html#mkdocs) accordingly.
+- Move your documentation from MkDocs and Markdown to Sphinx and reStructuredText. You can follow the steps we from the exercise for this. Create a new repository on GitHub for this called "sse-sphinx-exercise". When you are finished with this and/or need help create a new issue as described above, but name the issue `[USERNAME] Sphinx exercise`. Host your website on Read the Docs.
```

Comment:
```suggestion
- Move your documentation from MkDocs and Markdown to Sphinx and reStructuredText. You can follow the steps we know from the exercise for this. Create a new repository on GitHub for this called "sse-sphinx-exercise". When you are done with this and/or need help create a new issue as described above, but name the issue `[USERNAME] Sphinx exercise`. Host your website on Read the Docs.
```
I don't get the sentence

>  You can follow the steps we from the exercise for this.

I guess you mean the lecture?

Reply:
Which of the sentences?

Reply:
Ah...  I understand now... :smile: